indexers/subquery.md

```diff
@@ -0,0 +1,21 @@
+# SubQuery Indexer
+
+## Intro
+
+SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports 100+ ecosystems including OriginTrail, Polkadot, Cosmos, Ethereum, Polygon, Algorand, NEAR, and Avalanche.
+
+Another Subquery's competitive edge lies in the capacity to consolidate data, not just within a single blockchain but across multiple blockchains within a unified project. This facilitates the development of comprehensive dashboard analytics, cross-chain block scanners, or initiatives aimed at indexing XCM messages across parachains.
+
+Other advantages include superior performance with multiple RPC endpoint configurations, multi-worker capabilities and a configurable caching architecture. To find out more, visit our [documentation](https://academy.subquery.network/).
+
+**Useful resources**:
+- SubQuery Docs: [SubQuery Academy (Documentation)](https://academy.subquery.network/)
+- Intro Quick Start Guide: [1. Create a New Project](https://academy.subquery.network/quickstart/quickstart.html)
+- [OriginTrail Starter Project](https://github.com/subquery/subql-starter/tree/main/OriginTrail/origintrail-starter)
+
```

Comment:
Can you provide more information from the starter project here, how it can be used, not only useful resource and link to the starter project. Also it is important to point out that this is only indexing transfers from balances pallet, as OriginTrail community is generally more interested in the indexing of evm events on OriginTrail parachain, so some guides how to achieve that would be useful